### DISCUSSION TITLE
[Support]: Label count not increasing on Frigate+ dashboard

### DISCUSSION THREAD

**Discussion by jakeycrx** [blakeblackshear/frigate#15745] [General Support] [answered]
### Describe the problem you are having

Hi,

I haven't annotated any images for a couple of months now so I decided to go through a load of images. After a while I noticed that the label count isn't increasing for cats and dogs, although it does increase for car, person and face.

You can see it demonstrated in this video:

[Video](https://drive.google.com/file/d/1qV4YDW-Jfo8MraLrebhLLtn2BM9SI3DD/view?usp=sharing)

### Version

0.15.0-87e7b62

### What browser(s) are you using?

Google Chrome 131.0.6778.205

### Frigate config file

```yaml
N/A
```

### Relevant Frigate log output

```shell
N/A
```

### Relevant go2rtc log output

```shell
N/A
```

### FFprobe output from your camera

```shell
N/A
```

### Install method

HassOS Addon

### docker-compose file or Docker CLI command

```yaml
N/A
```

### Object Detector

Coral

### Network connection

Wired

### Camera make and model

Various

**Comment by NickM-27** [comment, marked as answer]
presumably it is because those cats were already submitted and saved, so the count there will not change. @blakeblackshear  can confirm

**Reply by jakeycrx** [reply]
You're right. If I submit an image from the Frigate UI to Frigate+ then the count increases as expected.
